### PR TITLE
Blog entry

### PR DESCRIPTION
**Pull request by cabarnes**
Adds models and API support for including questions and answers for programs. Questions are copied from the program being remixed. The question list has sequence numbers to order properly and a `required` parameter:
```
"blog_questions": [
{
  "id": 6,
  "question": "Question1",
  "answer": "Answer1",
  "sequence_number": 1,
  "required": true
},
{
  "id": 7,
  "question": "Question2",
  "answer": null,
  "sequence_number": 2,
  "required": true
},
{
  "id": 8,
  "question": "Question3",
  "answer": "Answer2",
  "sequence_number": 3,
  "required": false
}
],
```

### CODE REVIEW

File: mission_control/models.py [hunk 1]
```diff
@@ -48,3 +48,53 @@ class Tag(models.Model):
...
+    def __str__(self):
+        """Convert the model to a human readable string."""
+        return str(self.question)
+
+
+class BlockDiagramBlogQuestion(models.Model):
+    """Blog question for a block diagram."""
+
+    block_diagram = models.ForeignKey(
+        BlockDiagram, related_name='blog_questions', on_delete=models.CASCADE)
```
Comment (hbradio) [inline mission_control/models.py:67]: This means deleting the BlockDiagram will delete the BlockDiagramBlogQuestion, not the other way around, right?

**Review by hbradio — APPROVED** [review]

**Comment by hbradio** [comment]
Looks awesome!

File: mission_control/models.py (hunk 1 above)
Comment (cabarnes) [reply mission_control/models.py:67]: Correct

**Review by hbradio — APPROVED** [review]
:+1:

**Pull request merged**